tardis/tardis_portal/api.py

```diff
@@ -872,3 +884,20 @@ def hydrate(self, bundle):
             bundle.data['file_object'].close()
             del(bundle.data['file_object'])
         return bundle
+
+
+class ObjectAclResource(MyTardisModelResource):
+    content_object = GenericForeignKeyField({
+        Experiment: ExperimentResource,
+        # ...
+    }, 'content_object')
+
+    class Meta:
+        authentication = default_authentication
+        authorization = ACLAuthorization()
+        queryset = ObjectACL.objects.all()
+        filtering = {
+            'object_id': ('exact', ),
```

Comment:
That makes sense.  I think I was just translating "object_id" to "experiment_id" in my mind when using this filter to download an ObjectACL record for testing while learning how to use the API.  MyData doesn't actually use an "object_id" filter anywhere.  However it does create ObjectACLs with JSON which is currently missing the content_type field: https://github.com/monash-merc/mydata/blob/develop/mydata/ObjectAclModel.py#L31

So I think we do need the ContentTypeResource foreign key field in ObjectAclResource.  I'll try implementing that.

BTW, Carlo has a simple method (implemented for the Experiment type only) to fill in the content type field:

https://github.com/carlohamalainen/mytardis/blob/extend_rest_api/tardis/tardis_portal/api.py#L492